src/mappings.jl

```diff
@@ -69,3 +72,56 @@ end
 
 f01(x) = 1 .- exp.(.- x.^2)
 f01_inv(y) = sqrt.(.- log.(1 .- y))
+
+"""
+    Piecewise_positive()
+
+Returns a function and a inverse function inverse function
+to map numbers to larger than 0. 
+
+"""
+function Piecewise_positive()
+    return f02, f02_inv
+end
+
+f02(x) = ifelse.(x .> 0, one(eltype(x)) .+ x,one(eltype(x))./(one(eltype(x)).-x))
```

Comment:
Replace `one(eltype(x))` with `one(x)`.

Reply:
Your suggestion does not work, as far as I remember, this is why I changed it to eltype(x)